README.md

```diff
@@ -40,6 +40,13 @@ Clean, merge-ready pull requests:
       etc)
   - [ ] isolate platform-/environment-specific functionality
 
+##### Dependency Management
+  - [ ] avoids (or adequately justifies) the addition of 3rd-party code
+  - [ ] ensures the license is appropriate for inclusion
+  - [ ] the functionality is truly better-off as a dependency instead of a 1st-party implementation
+  - [ ] the 3rd-party code does not conflict with 1st-party standards
```

Comment:
> - [ ] the functionality is truly better-off as a dependency instead of a 1st-party implementation
> - [ ] the 3rd-party code does not conflict with 1st-party standards

These feel a bit more subjective—are there examples (or supporting articles / discussions) that we could provide to help clarify (1) when homegrown implementations should be used over off-the-shelf parts and (2) the extent to which code style should permeate through the dependency graph?